### PR TITLE
added common mistakes for junit topic

### PR DESCRIPTION
**Pull request by RomanDubovskyi**

### CODE REVIEW

File: java-core/junit/lambda-calc.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+### Common mistakes
+
+* Don’t ignore checkstyle rules.
+* Figure out which way is better to initialize your Calculator class instance: `@Before` or `@BeforeClass`, what’s the difference?
+* Make sure you name your methods according to [convention](https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names).
+* Add tests for each operation with different type of input params (positive values, negative values, zero etc) same goes for other operations.
+* Check if you’ve added all unneeded files to .gitignore before pushing.
```
Comment (boroda4436) [inline java-core/junit/lambda-calc.md:7]: а ми можемо для них вже зробити `.gitignore`  файл?

File: java-core/junit/lambda-calc.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+* You don’t need main method to see how your solution works, you have tests for this purpose.
+* If you use `delta` for comparison - declare it as a constant.
+* If you are expecting exception to be thrown in the test, you can do it this way:`@Test(expected = YourException.class)`.
```
Comment (boroda4436) [inline java-core/junit/lambda-calc.md:10]: Глянь відео, я ще показував там `assertThrows(...)`. Цей варіант я наче не показував

File: java-core/junit/lambda-calc.md (hunk 1 above)
Comment (RomanDubovskyi) [reply java-core/junit/lambda-calc.md:7]: у них там є .gitignore уже, можливо пушили ще додаткові файли, тому раніше був доданий цей пункт. В будь-якому випадку ще раз переглянув завдання - з цим проблем ніби не повинно виникнути, тому поки цей пункт прибрав

File: java-core/junit/lambda-calc.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+### Common mistakes
+
+* Don’t ignore checkstyle rules.
+* Figure out which way is better to initialize your Calculator class instance: `@Before` or `@BeforeClass`, what’s the difference?
```
Comment (boroda4436) [inline java-core/junit/lambda-calc.md:4]:
```suggestion
* Figure out which way is better to initialize your Calculator class instance: `@BeforeEach` or `@BeforeAll`, what’s the difference?
```
Згадав, що в JUnit 5 нові імена аннотацій

**Review by boroda4436 — APPROVED** [review]
